its/ruling/src/test/resources/expected/kotlin/kotlin-language-server/kotlin-S1186.json

```diff
@@ -0,0 +1,16 @@
+{
+'kotlin-kotlin-language-server-project:server/src/main/kotlin/org/javacs/kt/KotlinLanguageServer.kt':[
+164,
```

Comment:
I created [SONARKT-250](https://sonarsource.atlassian.net/browse/SONARKT-250) to fix this false-positive

Reply:
I left a comment on this one, as the RSPEC explicitly addresses this case already, indicating that this is a TP.